Gems/PhysX/Code/Tests/PhysXTestCommon.cpp

```diff
@@ -63,6 +64,16 @@ namespace PhysX
             }
         }
 
+        float CalculateZFromFalling(float gravityMultipler, float totalTimeSteps, float startingZPosition)
+        {
+            // Calculate distance fallen (d = 0.5 * g * t^2)
+            float gravity = startingZPosition * gravityMultipler;
```

Comment:
g in that equation is the acceleration due to gravity, which you can get from the GetGravity method in the scene interface (or in the tests you can just assume it's 0, 0, -10). startingZPosition should not affect the distance fallen. 

It won't apply to character controllers, but one nuance to be aware of for rigid bodies is that they have some linear damping by default, which means they fall a bit slower than you would expect just from gravity.

Reply:
I had already removed these from my local but forgot to push them up. I'll add this feedback to the notes of the backlog item.